src/frequenz/client/common/microgrid/components/__init__.py

```diff
@@ -16,10 +18,16 @@
 from frequenz.api.common.v1.microgrid.components.components_pb2 import (
     ComponentStateCode as PBComponentStateCode,
 )
+from frequenz.core.id import BaseId
 
 # pylint: enable=no-name-in-module
 
 
+@final
+class ComponentId(BaseId, str_prefix="CID"):
+    """A unique identifier for a microgrid component."""
+
+
```

Comment:
But definition ComponentId in client.microgrid  is different. The original version checks if component_id != 0.
Why Is this version different?

Reply:
IIRC that was a bogus limitation. @tiyash-basu-frequenz ?

Reply:
I remember it was there because grid has always id=0.
But I don't know if it is still true.